fastai/data_block.py

```diff
@@ -197,6 +197,20 @@ def random_split_by_pct(self, valid_pct:float=0.2, seed:int=None)->'ItemLists':
         cut = int(valid_pct * len(self))
         return self.split_by_idx(rand_idx[:cut])
 
+    def split_subsets(self, train_size:float, valid_size:float, seed=None) -> 'ItemLists':
+        """Split the items into train set with size `train_size * n` and valid set with size `valid_size * n`.
+
+        The total size can be smaller than 1 to work with a subset of the data.
+        """
+        assert 0 < train_size < 1
+        assert 0 < valid_size < 1
+        assert train_size + valid_size <= 1.
+        if seed is not None: np.random.seed(seed)
+        n = len(self)
```

Comment:
Small detail, but `len(self.items)` is safer since `len(self)` pretends to be 1 when it should be 0 sometimes (a hack to go with pytorch DataLoader).

Reply:
That's interesting. Done.